libraries/SITL/SITL.cpp

```diff
@@ -430,25 +430,49 @@ const AP_Param::GroupInfo SIM::var_ins[] = {
     AP_GROUPINFO("IMUT_TCONST",   3, SIM, imu_temp_tconst, 300),
     AP_GROUPINFO("IMUT_FIXED",    4, SIM, imu_temp_fixed, 0),
     AP_GROUPINFO("ACC1_BIAS",     5, SIM, accel_bias[0], 0),
+#if INS_MAX_INSTANCES > 1
```

Comment:
don't we need to include #AP_InertialSensor.h ?

Reply:
Yep - added it.

Really we should be defining an `AP_SIM_INS_MAX_INSTANCES` but let's not go there today :-)